core/selection-controller.js

```diff
@@ -0,0 +1,31 @@
+/**
+ * @module core/selection-controller
+ */
+var Montage = require("montage").Montage;
+
+/**
+ * @class SelectionController
+ * @extends external:Montage
+ */
+exports.SelectionController = Montage.specialize(/* @lends SelectionController.prototype */ {
+    constructor: {
+        value: function SelectionController() {
+            this.super();
+        }
+    },
+
+    /**
+     * @type {boolean}
+     */
+    isMultipleSelectionAllowed: {
+        value: false
+    },
+
+    /**
+     * @type {[object]}
+     */
+    selection: {
+        value: []
+    }
+
+});
```

Comment:
It may be worth testing the JSDoc generation... I think "[object]" should be "object[]", but in any case we should test JSDoc generation so we don't check in code that breaks the JSDocs.